2023/opmodes/javabotsLM1Auto.java

```diff
@@ -228,23 +272,19 @@ public void handleEvent(RobotEvent e) {
         });
     }
 
-    public void delay(int seconds)
+    public void delayAndLowerLift(int seconds)
     {
-        this.addTask(new SingleShotTimerTask(this, 1000*seconds) {
+        addTask(new SingleShotTimerTask(this, 1000*seconds) {
```

Comment:
I'm going to assume that 'seconds' here aren't really seconds because 1000 of them is 16+ minutes and you are multiplying that by the seconds parameter passed in, which if '2', for example, would be over half an hour.   Which implies that the parameter's unit that is being passed in, perhaps, is milliseconds?  Which would suggest that the parameter name should be changed to prevent confusion.